frappe/handler.py

```
@@ -122,8 +122,6 @@ def uploadfile():
 				is_whitelisted(method)
 				ret = method()
 	except Exception:
```

@vjFaLk @Alchez I'm not sure if this can be removed. This leaves a server traceback.

---

Yeah no, you should not remove that.

---

Why?

---

@vjFaLk the issue here is in case you upload an incorrect filetype, the system throws a server error due to this code. it must only tell the user to upload the correct file type

---

I understand, but this is a generic handler for any kind of file type. Suppressing the error here would mean that if there's a valid error, it'll end up not being thrown.